model/stories.go

```diff
@@ -1,39 +1,29 @@
 package model
 
 import (
-	storyviews "github.com/source-academy/stories-backend/view/stories"
 	"gorm.io/gorm"
 )
 
 var DB *gorm.DB
 
-type StoryDB struct {
+type Story struct {
 	gorm.Model
-	AuthorID uint
-	Content  string
+	AuthorID     uint
+	StoryContent string
```

Comment:
Was there any reason this was changed? This should be `Content` to match the column name in the migration file.